TECHMANIA/Assets/Scripts/DiscordController.cs

```diff
@@ -0,0 +1,33 @@
+using System;
+using UnityEngine;
+
+public class DiscordController : MonoBehaviour
+{
+#if !UNITY_IOS && !UNITY_ANDROID
+    private static Discord.Discord discord;
+
+    public static void Start ()
+    {
+        if (discord != null) return;
+        discord = new Discord.Discord(802017593086836767, (System.UInt64)Discord.CreateFlags.Default);
```

Comment:
Is `802017593086836767` some sort of secret? Is it okay to publish it into the repo?

Reply:
It is Discord application ID, I think it's safe to put it here.